docs/source/en/index.md

```diff
@@ -254,6 +254,8 @@ Flax), PyTorch, and/or TensorFlow.
 |                        [PVTv2](model_doc/pvt_v2)                         |       ✅        |         ❌         |      ❌      |
 |                       [QDQBert](model_doc/qdqbert)                       |       ✅        |         ❌         |      ❌      |
 |                         [Qwen2](model_doc/qwen2)                         |       ✅        |         ❌         |      ❌      |
+|                   [Qwen2Audio](model_doc/qwen2_audio)                    |       ✅        |         ❌         |      ❌      |
+|            [Qwen2AudioEncoder](model_doc/qwen2_audio_encoder)            |       ✅        |         ❌         |      ❌      |
```

Comment:
I don't think we should expose Qwen2AudioEncoder as it is a part of the model, and not usable by itself. Let's not expose it

Reply:
I think you can add it to [`PRIVATE_MODELS`](https://github.com/faychu/transformers/blob/066146fd43f07f024de63e17a5bbda4a190652c1/utils/check_repo.py#L62), but I'd like to double-check with @amyeroberts.

Another solution would be to remove `qwen2_audio_encoder` from the `configuration_auto` and `modeling_auto` classes, but then you wouldn't be able to load the config as you do

Reply:
Yes, adding to `PRIVATE_MODELS` is the way to go if you want to have a separate config. We do this for e.g. CLIP's text model so OK imo

Reply:
Thanks, already fixed it [here](https://github.com/huggingface/transformers/pull/32137/commits/1a86c116737f9c778dba77c4a6136c40170f7786).